doc/mgmt/SONiC_OC_Interface_HLD.md

```diff
@@ -586,6 +648,31 @@ Mode of IPv4 address assignment: MANUAL
 IPv6 address is a::b/64
 Mode of IPv6 address assignment: MANUAL
 ```
+#### Display interface COUNTERS
+`show interface counters`
+Units: RX_MBPS/TX_MBPS(MB/s), RX_MbPS|TX_MbPS(Mb/s), RX_PPS|TX_PPS(pkts/s)
+
+Interface    State    RX_OK RX_MBPS  RX_MbPS  RX_PPS  RX_UTIL    RX_ERR    RX_DRP    RX_OVR    TX_OK  TX_MBPS    TX_MbPS  TX_PPS  TX_UTIL    TX_ERR    TX_DRP    TX_OVR
+------------  -------  -------  --------  --------  ---------  --------  --------  --------  -------    --------  --------  ---------  --------  --------  --------
+  Ethernet48   U     87,219  0.00   0.00  0.00        0.00%         0         3         0   86,871      0.00   0.00  0.00 0.00%         0         0         0
+  Ethernet49   U    5,649  0.00   0.00  0.00 0.00%         0     5,649         0    5,645      0.00   0.00  0.00 0.00%         0         0         0  
+  ........
+  PortChannel1 U    5,649  0.00   0.00  0.00 0.00%         0     5,649         0    5,645      0.00   0.00  0.00 0.00%         0         0         0
+```
+##### show interface Ethernet <>
+```
+sonic# show interface Ethernet 64
+Ethernet64 is up, line protocol is up
+Hardware is Eth
+Mode of IPV4 address assignment: not-set
+Mode of IPV6 address assignment: not-set
+Interface IPv6 oper status: Disabled
+IP MTU 9100 bytes
+LineSpeed 25GB, Auto-negotiation off
+Last clearing of "show interface" counters: 1970-01-01 00:00:00
+30 seconds input rate 84640 bits/sec, 10236 Bytes/sec, 52 packets/sec
```

Comment:
Can we have packets/sec first followed by bits/sec?
30 seconds input rate 52 packets/sec, 84640 bits/sec, 10236 Bytes/sec

Generally most of the vendors follow this order and hence the ask

Reply:
Addressed.